docs/user_manual.md

```diff
@@ -588,3 +571,42 @@ The name of a cluster is immutable, and it is included in the names of all of th
+
+The locking system uses the `instanceIDPrefix` from the cluster spec to identify an instance of the operator. Make sure to set this to a unique value for each Kubernetes cluster, both to support the locking system and to prevent duplicate instance IDs.
+
+This locking system uses the FoundationDB cluster as its data source. This means that if the cluster is unavailable, no instance of the operator will be able to get a lock. If you hit a case where this becomes a blocker, you can disable the locking system by setting `lockOptions.disableLocks = true` in the cluster spec.
+
+In most cases, bounces will be done independently in each Kubernetes cluster, and the locking system will be used to ensure a minimum time between the different bounces and avoid multiple recoveries in a short span of time. During upgrades, however, all instances must be bounced at the same time. The operator will use the locking system to coordinate this. Each instance of the operator will store records indicating what processes it is managing and what version they will be running after the bounce. Each instance will then try to acquire a lock and confirm that every process reporting to the cluster is ready for the upgrade. If all processes are prepared, the operator will bounce all of them at once. If any instance of the operator is stuck and unable to prepare its processes for the upgrade, the bounce will not occur.
+
+## Deny List
+
+There are some situations where an instance of the operator is able to get locks but should not be trusted to perform global actions. For instance, the operator could be partitioned in a way where it cannot access the Kubernetes API but can access the FoundationDB cluster. To block such an instance from taking locks, you can add it to the `denyList` in the lock options. You can set this in the cluster spec on any Kubernetes cluster.
+
+    apiVersion: apps.foundationdb.org/v1beta1
+    kind: FoundationDBCluster
+    metadata:
+      name: sample-cluster
+    spec:
+      instanceIDPrefix: dc1
+      lockOptions:
+        denyList:
+          - id: dc2
+
+This will clear any locks held by `dc2`, and prevent it from taking further locks. In order to clear this deny list, you must change it to allow that instance again:
```

Comment:
Is there a reason just removing the instance is not good enough? Since the name `denyList` list implies that any instance on this list is not allowed to take locks.

Reply:
Yes. We can talk about that more in the Deny List PR.